index.js

```diff
@@ -128,9 +128,22 @@ exports.createMVPDataFile = function(performanceYear) {
       fs.readFileSync(path.join(__dirname, 'mvp', performanceYear.toString(), 'mvp.json')));
     measuresData = this.getMeasuresData(performanceYear);
   } catch (e) {
-    console.log('QPP measures data not found for year: ' + performanceYear + ' --> ' + e);
+    console.log('QPP mvp / measures data not found for year: ' + performanceYear + ' --> ' + e);
   }
 
+  const mvpIds = [];
+  mvpData.forEach(mvpDataItem => {
+    if (mvpDataItem.mvpId !== 'app1') {
```

Comment:
`app1` is already present in the measures-data.json file, so we don't want to remove it